webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js

```diff
@@ -131,6 +140,133 @@ class SakaiRubricsManager extends RubricsElement {
       });
     });
   }
+
+  sortSiteRubrics(event) {
+    var sortInput = event.target.className;
+    var sortInputValue = sortInput.toString().toLowerCase();
+    var ascending = true; 
+    //icon from header
+    $('.sort_element_site').each( function( i, val ) {
+      var className = $( val ).attr('class');
+      if (!className.includes(sortInputValue)) {
+        $( val ).removeClass('collpase-icon fa fa-chevron-down');
+        $( val ).removeClass('collpase-icon fa fa-chevron-up');
```

Comment:
You should write this using vanilla js.
this.querySelectorAll(".sort_element_site").forEach(val => {
  const className = val.getAttribute("class");
  if (!className.includes(sortInputValue)) {
    val.classList.remove("collapse-icon", "fa", "fa-chevron-down");
});

You get the idea ...
I want to steadily remove the use of jquery if there's a suitable alternative in vanilla js. The less dependency on jquery, the better. Also, if you have to use jquery, you should not keep wrapping an Element in a jquery object as it's really inefficient. Just use the Element if you can :)

I also have one more concern here. Why do you need to remove these classes in the first place? Can you not avoid adding them, in the lit-html template from the render method for that component?